### PR TITLE
feat: Support plotting of Timestream objects

### PR DESCRIPTION
**Pull request by bjchambers**

### CODE REVIEW

File: python/pysrc/kaskada/plot.py
Comment (jordanrfrazier) [inline python/pysrc/kaskada/plot.py:1]: Should these live in a different directory? docs, visualizations, ?

File: python/pytests/shift_by_test.py [hunk 1]
```diff
@@ -10,9 +10,9 @@ def source() -> kd.sources.CsvString:
         [
             "time,key,m,n",
             "1996-12-19T16:39:57,A,5,10",
-            "1996-12-19T16:39:58,B,24,3",
-            "1996-12-19T16:39:59,A,17,6",
-            "1997-01-18T16:40:00,A,,9",
+            "1997-02-19T16:39:58,B,24,3",
+            "1997-03-19T16:39:59,A,17,6",
+            "1997-008-18T16:40:00,A,,9",
```
Comment (jordanrfrazier) [inline python/pytests/shift_by_test.py:15]: These feel like unintentional changes?

**Review by jordanrfrazier — APPROVED** [review]

File: python/pysrc/kaskada/_timestream.py [hunk 2]
```diff
@@ -124,6 +124,11 @@ def data_type(self) -> pa.DataType:
         """The PyArrow type of values in this Timestream."""
         return self._ffi_expr.data_type()
 
+    @property
+    def is_continuous(self) -> bool:
+        """Returns true if this Timestream is continuous."""
```
[the remark this answers is not in the document]
Comment (bjchambers) [reply python/pysrc/kaskada/_timestream.py:129]: Yup! And mostly the user shouldn't care -- but this is useful for visualization, etc.

File: python/pysrc/kaskada/plot.py
Comment (bjchambers) [reply python/pysrc/kaskada/plot.py:1]: I actually think it's good here -- this let's the user do `kd.plot.render(...)` to render a plot. As mentioned at standup, we could even allow `timestream.preview_plot()` or something. Useful while developing things in a notebook.

File: python/pytests/shift_by_test.py (hunk 1 above)
Comment (bjchambers) [reply python/pytests/shift_by_test.py:15]: Yes.

File: python/pysrc/kaskada/_timestream.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (bjchambers) [reply python/pysrc/kaskada/_timestream.py:129]: (By care I mean use this method)

**Pull request merged**